pkg/acceptance/cli_test.go

```diff
@@ -32,6 +32,7 @@ const containerPath = "/go/src/github.com/cockroachdb/cockroach/cli/interactive_
 var cmdBase = []string{
 	"/usr/bin/env",
 	"COCKROACH_SKIP_UPDATE_CHECK=1",
+	"COCKROACH_CRASH_REPORTS=''",
```

Comment:
nit that you can feel free to ignore: `COCKROACH_CRASH_REPORTS=` (i.e., no quotes) is entirely equivalent.

Reply:
I like the empty quotes

Reply:
nope

Reply:
heh, fwiw, that was a "nope I don't feel strongly at all"

Reply:
I needed another CI cycle anyway